src/Importer/JsonResourceImporter.php

```diff
@@ -38,7 +38,13 @@ public function import(string $fileName): ImporterResultInterface
             throw new ImporterException(sprintf('File %s could not be loaded', $fileName));
         }
 
-        foreach (json_decode($contents, true) as $i => $row) {
+        $data = json_decode($contents, true);
+
+        if (is_null($data)) {
```

Comment:
You need to write
```suggestion
        if (null === $data) {
```

for fix Ecs error in Travis.